README.md

```diff
@@ -452,7 +517,7 @@ let _ : unit Js.Promise.t =
 (* Chaining *)
 let _ : unit Js.Promise.t =
   Js.Promise.then_
-  	(fun value -> Js.Promise.resolve (Js.log value))
+    (fun value -> Js.Promise.resolve (Js.log value))
```

Comment:
This isn't your doing, but I just noticed how weird it is to resolve the return value of `Js.log`. And being more explicit with `Js.log value; Js.Promise.resolve ()` isn't much more verbose either. What do you think?

Reply:
Both ok. `Js.Promise.resolve (Js.log value)` is less code. I think everyone knows the return value from `Js.log` is `unit`.